MetaMask/metamask-extension
# Fix Unable to determine contract standard error

## Explanation

Inside the **useTransactionDisplayData** hook we were sending the wrong token address to the **getAndSetAssetDetails** function and that would cause error : **Unable to determine Contract Standard**

This PR will ensure to send correct token address.




Fixes #18135 
Fixes #17287 


## Manual Testing Steps



- Go to test dapp https://metamask.github.io/test-dapp/
- Deploy Collectibles
- Mint a collectible
- Add collectible to Wallet
- Click NFT details
- Send NFT
- Confirm -- check MetaMask UI console
- Notice there is no warning from the bug ticket

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
